src/msg/async/ProtocolV1.cc

```diff
@@ -2052,7 +2018,7 @@ CtPtr ProtocolV1::handle_connect_message_2() {
                                       authorizer_reply);
   }
 
-  ceph::buffer::list auth_bl_copy = authorizer_buf;
+  ceph::bufferptr auth_bp_copy = authorizer_buf;
```

Comment:
nit: `buffer::ptr` (and `buffer::list` respectively) to preserve the convention.